sklearn/tests/test_calibration.py

```diff
@@ -275,6 +275,34 @@ def multiclass_brier(y_true, proba_pred, n_classes):
     assert calibrated_brier < 1.1 * uncalibrated_brier
 
 
+def test_calibration_zero_probability():
```

Comment:
Maybe add a comment to explain that this test checks an edge case where `_CalibratedClassifier` avoids numerical errors in the multiclass normalization step if the all the calibrators output zero all at once for a given sample and instead fallback to uniform probabilities.

Reply:
Thanks, I added comment to this test.